### PR TITLE
Us s4 content article + update CSS

### PR DESCRIPTION
**Pull request by pwah59**

### CODE REVIEW

**Review by alexboutemy — APPROVED** [review]

File: src/app/components/paragraph/paragraph.component.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
+@use "/src/_import.scss" as *;
+
+.comment {
```
Comment (FlorianHe) [inline src/app/components/paragraph/paragraph.component.scss:3]: Comment in paragraph?

File: src/app/components/paragraph/paragraph.component.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div *ngIf="paragraph" class="paragraph">
```
Comment (FlorianHe) [inline src/app/components/paragraph/paragraph.component.html:1]: why not adding the CSS_PARAGRAPH[paragraph.typeContent] class on the first div (and make the div a section)

File: src/app/components/paragraph/paragraph.component.scss
Comment (FlorianHe) [inline src/app/components/paragraph/paragraph.component.scss:1]: Remove all useless comment class call + encapsulate depending of the CSS_PARAGRAPH class

File: src/app/pages/article/article.component.ts [hunk 3]
```diff
@@ -20,6 +20,8 @@ export class ArticleComponent implements OnInit {
       this.articleService.getArticleById(params['id'])
         .subscribe(article => {
           this._article = article;
+          // Temporaire le temps de merge les users pour éviter une erreur
```
Comment (Benja-san) [inline src/app/pages/article/article.component.ts:23]: put this comment on TODO in order to keep it in mind (and in eyes)

File: src/app/components/paragraph/paragraph.component.html (hunk 2 above)
Comment (Benja-san) [reply src/app/components/paragraph/paragraph.component.html:1]: would be nice indeed

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Review by Benja-san — APPROVED** [review]

**Review by alexboutemy — DISMISSED** [review]

**Review by alexboutemy — APPROVED** [review]

**Review by FlorianHe — APPROVED** [review]

**Pull request merged**